src/Makefile.OCaml

```diff
@@ -51,6 +51,10 @@ OCAMLLIBDIR:=$(shell $(OCAMLC) -config-var standard_library)
 #
 OCAMLFIND := $(shell command -v ocamlfind 2> /dev/null)
 
+# Any invocation of the non-Xcode version of xcodebuild will trigger an error message
+# and an non 0 exit code. 
+XCODEBUILD := $(shell xcodebuild -version > /dev/null; echo $$?)
+
```

Comment:
Is this being run on non-mac?  (It shouldn't be.)

Reply:
Is it needed to assign `XCODEBUILD` at all? Just have the `$(shell)` invocation directly in the `ifeq`.

Reply:
> Is this being run on non-mac? (It shouldn't be.)

Ok thanks I will move the check in the case where we are in `Darwin`, `ifeq ($(OSARCH), Darwin) ...`